File: app/src/main/java/no/nav/tilbakemeldingsmottak/consumer/oppgave/domain/HentOppgaveResponseTo.java
package no.nav.tilbakemeldingsmottak.consumer.oppgave.domain;

import lombok.Builder;
import lombok.Value;

@Value
@Builder
public class HentOppgaveResponseTo {
    String id;
    String aktoerId;
    String orgnr;
    String tildeltEnhetsnr;
    String tema;
    String versjon;
    String aktivDato;
    String prioritet;
    String oppgavetype;
    String journalpostId;
    String status;
}

File: app/src/main/kotlin/no/nav/tilbakemeldingsmottak/consumer/oppgave/domain/HentOppgaveResponseTo.kt
package no.nav.tilbakemeldingsmottak.consumer.oppgave.domain

data class HentOppgaveResponseTo(
    val id: String,
    val tildeltEnhetsnr: String,
    val tema: String,
    val oppgavetype: String,
    val versjon: String,
    val prioritet: String,
    val status: String,
    val aktivDato: String,

    val aktoerId: String? = null,
    val orgnr: String? = null,
    val journalpostId: String? = null,
)
